lib/produce.csv

```diff
@@ -0,0 +1,535 @@
+name,category,country_code,month_index
+abacaxi havaí,fruit,BR,1

```

Comment:
More as a thing for the future, but we should considering having these names in a slug format (and maybe in English, not sure how feasible that is).

So instead of `abacaxi havaí`, we'd have `abacaxi_havai`. And then we could add it to our i18n files.

Reply:
Adding the slugs is easy, however the names in english I'm not so sure, as there are some local produce as well like cupuaçu, and the different types of mangos or pineapples etc Eg, would we translate abacaxi_havai to pineapple_havai?

I thought about dropping the types, but they are in season in different moments, and in some cases, the types do make a difference, like tangerina murcot vs tangerina cravo

Reply:
No idea about translations, also that's not something we should worry about now, it looks a bit too early. Let's keep as is :)